.github/workflows/qa.yml

```diff
@@ -1,26 +1,113 @@
 name: Q&A
 
 on:
-  pull_request:
+    pull_request:
 
 defaults:
-  run:
-    shell: bash
+    run:
+        shell: bash
 
 jobs:
 
-  tests:
-    name: Checks
-    runs-on: Ubuntu-20.04
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: No symlinks
-        run: |
-          SYMLINKS=$(find -type l)
-          if [[ "" != "$SYMLINKS" ]]; then
-            echo "::error::Symlinks are not allowed"
-            echo "Found $SYMLINKS"
-          fi
+    tests:
+        name: Checks
+        runs-on: Ubuntu-20.04
+
+        steps:
+            -
+                name: Checkout
+                uses: actions/checkout@v2
+                id: checkout
+
+            -
+                name: Install tools
+                run: |
+                    (cd .github/tools && composer install --no-dev)
```

Comment:
This tool relies on PHP, so you should use the `setup-php` action

Reply:
there is no need to, all works well already